### PR TITLE
Fix company logo overwrite

### PR DESCRIPTION
**Pull request by jekkos**
Potential fix for #3652

### CODE REVIEW

**Comment by objecttothis** [comment]
Reviewing now.

File: app/Controllers/Config.php [hunk 1]
```diff
@@ -341,6 +341,7 @@ public function postSaveInfo(): void
 
 		if(!empty($upload_data['orig_name']) && $upload_data['raw_name'])
 		{
+			$filename = md5(date('Y-m-d H:i:s:u'));
```
Comment (objecttothis) [inline app/Controllers/Config.php:344]: Looks like you're assigning `$filename` here, but I don't see where you are using it.

File: app/Controllers/Items.php [hunk 2]
```diff
@@ -845,11 +845,10 @@ private function upload_image(): array
 		}
 
 		$filename = $file->getClientName();
-		$info = pathinfo($filename);
 
 		$file_info = [
 			'orig_name' => $filename,
-			'raw_name' => $info['filename'],
+			'raw_name' => md5(date('Y-m-d H:i:s:u')),
```
Comment (objecttothis) [inline app/Controllers/Items.php:851]: Did you decide against using a hash of filename + datestamp? It's not in a foreach/while loops, so I guess the only way for a duplicate is if two different sessions ran this line in the same microsecond. Not a likely scenario, but not impossible. I don't feel strongly about it.

**Review by objecttothis — CHANGES_REQUESTED** [review]

**Comment by jekkos** [comment]
Comments were addressed but I shuold do a short retest

**Comment by jekkos** [comment]
Comments addressed , tested locally and seems fine, will merge.

**Pull request merged**